<launch>

 <!-- Pubblica la trasformazione relativa fra il tool e la ventosa -->
  	<node pkg="tf2_ros" type="static_transform_publisher" name="tip_broadcaster" args="0.0 0.0 0.211 -0.5 -0.5 0.5 0.5 tool tool_extremity" />
  	<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="tip_broadcaster" args="0.069 -0.292 0.050 -0.032 0.514 0.019 0.857 right_hand_ee_link tool_extremity" /> -->
 <!-- Pubblica la posa di grasp rispetto al frame tool -->
  	<node pkg="tf2_ros" type="static_transform_publisher" name="grasp_broadcaster" args="0.067 0.011 -0.048 0.111 0.700 -0.695 -0.116 tool hand_pose_tool" />

 	<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="grasp_broadcaster" args="0.071 -0.007 -0.035 -0.164 -0.653 0.724 0.151 tool hand_pose_tool" /> -->
	<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="grasp_broadcaster" args="0.003 0.045 -0.014 0.633 -0.242 -0.559 0.478 tool hand_pose_tool" /> -->
	<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="grasp_broadcaster" args="-0.060 0.066 -0.016 0.671 -0.192 -0.264 0.666 tool hand_pose_tool" /> -->

 <!-- Pubblica la trasformazione relativa fra il frame base del robot e l'oggetto da prelevare -->

 	<!-- Parametri oggetti da prendere -->
	<param name = "n_object" value="1" />
	<param name = "object_name_1" value = "box1" />
  	<param name = "object_name_2" value = "box3" />
  	<param name = "object_name_3" value = "box5" />

    <node pkg = "script_pkg" name = "object_pose" type = "object_pose.py" output = 'screen'/>  -->

    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="fake_tool" args="0.112 0.374 0.792 0.913 -0.407 -0.028 -0.012 robot_arm_link0 fake_tool" />  -->

</launch>
